addons/mail/models/res_partner.py

```diff
@@ -20,6 +22,15 @@ class Partner(models.Model):
     vat = fields.Char(tracking=5)
     # channels
     channel_ids = fields.Many2many('mail.channel', 'mail_channel_member', 'partner_id', 'channel_id', string='Channels', copy=False)
+    # add tracked field
+    contact_address_inline = fields.Char(compute='_compute_contact_address_inline', string='Inlined Complete Address', tracking=True)
```

Comment:
Tracking an unstored field seems strange to me ^^ . It works a bit by side-effects (computed has an initial value, changing its triggers make a new computation and it triggers a tracking I suppose), but I fear that it may break in case of change ...

The other way round would be to call "_mail_track" manually, but we could have to check any address field updated at create / write and call the tracking methods. All in all it would lead to the same with more code ... so hesitating. I suppose we could move forward with it, would just add a test (on partner model, like in test_res_partner.py) to be sure we generate one message / flush (and if someday unstored fields tracking is broken it would be highlighted ^^ ) .

Reply:
Mm, yes I suppose it's more of an implementation detail that it works. It's 'just' because we use the ORM to check the value before and after, so it's computed instead of being `None`.

I feel like it makes 'functional' sense though, there may be other instances where you might want to say 'hey, changing this will also change the result of that!'

Will add a test for this specific case :)